src/Build/BackEnd/Components/SdkResolution/SdkResolverServiceException.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using Microsoft.Build.Framework.BuildException;
+
+#nullable disable
```

Comment:
nit: Is there reason for disabling nullability checks in this file?

Reply:
oh, apparently, there was a reason for that

`Possible null reference argument for parameter 'inner' in 'SdkResolverServiceException.SdkResolverServiceException(string message, Exception inner)'.` that is needed by BuildExceptionSerializationHelper